package.json

```
@@ -0,0 +1,74 @@
+{
+  "name": "redux-blog",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "test": "jest --verbose",
```

Technically eslint is not getting run
```suggestion
    "test": "jest --verbose",
    "pretest": "npm run lint",
```